libres/lib/res_util/block_fs.cpp

```diff
@@ -864,15 +864,14 @@ static void block_fs_preload(block_fs_type *block_fs) { return; }
 */
 
 static void block_fs_fix_nodes(block_fs_type *block_fs,
-                               long_vector_type *offset_list) {
+                               const std::vector<long> &offset_list) {
     if (block_fs->data_owner) {
         fsync(block_fs->data_fd);
         {
             char *key = NULL;
-            for (int inode = 0; inode < long_vector_size(offset_list);
-                 inode++) {
+            for (int inode = 0; inode < offset_list.size(); inode++) {
```

Comment:
Consider: `for (auto &node_offset : offset_list)`?

Reply:
Changed.